src/devices/Media/SoundDevice/SoundConnectionSettings.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Media
+{
+    /// <summary>
+    /// The connection settings of a sound device.
+    /// </summary>
+    public class SoundConnectionSettings
```

Comment:
this seems to be mixing recording and playback settings. I'd recommend to either unify options (i.e. just DeviceName) or split classes.

Reply:
In the case of multiple sound card, it is possible to set device name. And I don't really want to separate the settings. For example, there's no recording device on the Raspberry Pi, and if I use SoundDevice, it's going to be complicated.